src/REST/keywords.py

```diff
@@ -1355,9 +1402,8 @@ def _instantiate(self, request, response, validate_schema=True):
                 self._validate_schema(request_properties, request)
             if response_properties:
                 self._validate_schema(response_properties, response)
-        request_properties["body"] = self._new_schema(request["body"])
-        request_properties["query"] = self._new_schema(request["query"])
-        response_properties["body"] = self._new_schema(response["body"])
+        self._form_sub_schema(request_properties, request)
```

Comment:
@asyrjasalo @asimell Sure, you can opt out my "Last REST Instance" keyword; but one change here that you might need to take into consideration is that before my changes only request & response body were outputted, but with my changes every key/value pair in the request & response are now outputted via "Last REST Instance" keyword.

Also, the end user of the library would have to write & maintain a function containing at least 10 lines of code, while 
the library itself has the instance data already ready to be written into a file. Imagine that tens of users does the same. Isn't that a wasteful effort from the end users' perspective? 

I have nothing further to say. Thanks anyway bringing life back to this important library.

Reply:
> only request & response body were outputted

Incorrect. If you call `Output` without any parameters it outputs the full request and response object.

There's also `Output Schema` to output the schema only and you can give a `file_path` argument to that keyword as well.